packages/skin/dist/card/card.css

```diff
@@ -0,0 +1,157 @@
+.card {
+    background-color: var(--color-background-secondary);
+    border: none;
+    border-radius: var(--border-radius-150);
+    color: var(--color-foreground-primary);
+    display: flex;
+    flex-direction: column;
+    max-width: 400px;
+    min-width: 200px;
+    outline: 6px solid transparent;
+    outline-offset: 4px;
+    padding-block-end: var(--spacing-150);
+    transition:
+        outline-width var(--motion-duration-short-2)
+            var(--motion-easing-quick-enter),
+        outline-color var(--motion-duration-short-2)
+            var(--motion-easing-quick-enter),
+        outline-offset var(--motion-duration-short-2)
+            var(--motion-easing-quick-enter);
+}
+
+.card--outlined {
+    background-color: inherit;
+    border: 1px solid var(--color-stroke-subtle);
+}
+
+a.card {
+    color: inherit;
+    text-decoration: none;
+}
+
+button.card {
+    font-size: inherit;
+    line-height: inherit;
+    padding: 0;
+    padding-block-end: var(--spacing-150);
+    text-align: start;
+    transition: all var(--motion-duration-medium-3)
```

Comment:
I think we should avoid using `all` and use the specific property that we want to transition. It might not impact now that only `filter` and `opacity` is being used, but in case in the future we start adding more properties that change it might impact performance. For example, toggling horizontal class will change other properties of `.card`

Reply:
@HenriqueLimas , good callout, actually. We were only transitioning the outline properties, but outline needs to be separated and start with no opacity. 

I swapped to use animation, which will specifically target the things animating. Thanks!